roles/adoption_osp_deploy/tasks/overcloud_deploy.yml

```diff
@@ -0,0 +1,58 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Deploy 17.1 overcloud
+  vars:
+    _architecture_path: "{{ cifmw_architecture_repo }}/automation/vars"
+    _roles_file: >-
+      {{
+        [_architecture_path,
+         _cifmw_scenario_content.adoption.roles_file
+        ] | path_join
+      }}
+    _network_data_file: >-
+      {{
+        [_architecture_path,
+         _cifmw_scenario_content.adoption.network_data_file
+        ] | path_join
+      }}
+    _overcloud_services_file: >-
+      {{
+        [_architecture_path,
+         _cifmw_scenario_content.adoption.overcloud_services_file
+        ] | path_join
+      }}
+    _overcloud_deploy_cmd: >-
```

Comment:
I'd be really more flexible here about what we can pass. Reduce the default parameters to a bare minimum, and let others pass whatever they need from the automation content.

Reply:
sounds good, I'll move more of them to the architecture vars files